### DISCUSSION TITLE
How to apply external stylesheet

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15638] [How to] [answered]
Folllowing some previous threads: How to apply external stylesheet to email templates?

Regards

**Comment by s2b-git** [comment]
What I mean is getting import or include (etc) but code is stripped even config got fixed $purifierAllowedDomains etc

**Reply by mariuszkrzaczkowski** [reply]
give sample styles

**Reply by JohnMaverick** [reply]
for instance In my case I need to turn certain text string to font awesome item, or certain check box field turn to mentioned. Maybe getting external style could do that? s2b got good point to allowing external styles to templates plus full html body/head etc?

**Reply by mariuszkrzaczkowski** [reply]
which type are we talking about:
Inline method
Nested sheet
Outer sheet

**Reply by s2b-git** [reply]
Outer? Called from different domain?

**Reply by JohnMaverick** [reply]
Or css withing application directory...

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
PDF has been rebuilt for this I improved HTML cleanup, check it out https://github.com/YetiForceCompany/YetiForceCRM/commit/397bab35727437121ab1d6b958e4f627421de35e
and try it:
`<link href="https://...." rel="stylesheet">
	`

**Reply by waran70** [reply]
how about emails templates? would it work?

**Reply by mariuszkrzaczkowski** [reply]
I did not try
